python/ray/rllib/models/modelv2.py

```diff
@@ -0,0 +1,153 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from ray.rllib.models.model import restore_original_dimensions
+
+
+class ModelV2(object):
```

Comment:
This should probably implement methods like `variables`, `trainable_variables`, etc. that `keras.Model` and `tf.Module` provide.